src/components/draggable/draggable-item.component.tsx

```diff
@@ -1,12 +1,30 @@
 import React from "react";
 import { useDrop, useDrag } from "react-dnd";
-import PropTypes from "prop-types";
-import styledSystemPropTypes from "@styled-system/prop-types";
 
 import { filterStyledSystemPaddingProps } from "../../style/utils";
-import { StyledDraggableItem } from "./draggable-item.style";
+import {
+  StyledDraggableItem,
+  StyledDraggableItemProps,
+  StyledIcon,
+} from "./draggable-item.style";
 
-const paddingPropTypes = filterStyledSystemPaddingProps(styledSystemPropTypes);
+export interface DraggableItemProps extends StyledDraggableItemProps {
```

Comment:
I think we should just extend `PaddingProps` here, this approach will add `isDragging` to the public interface